docs/subnets/reference-elastic-subnets-parameters.md

```diff
@@ -0,0 +1,224 @@
+<!-- vale off -->
+$$
+Reward = \left(MaximumSupply - Supply \right) \times \frac{Stake}{Supply} \times \frac{Staking Period}{Minting Period} \times EffectiveRate
+$$
+where
+$$
+EffectiveRate = \frac{MinConsumptionRate}{PercentDenominator} \times \left(1- \frac{Staking Period}{Minting Period}\right) + \frac{MaxConsumptionRate}{PercentDenominator} \times \frac{Staking Period}{Minting Period}
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+$Effective\:Period$ is a linear combination of $MinConsumptionRate$ and
+$MaxConsumptionRate$.
+$MinConsumptionRate$ and $MaxConsumptionRate$ bound $Effective\:Period$ because 
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+MinConsumptionRate \leq EffectiveRate \leq MaxConsumptionRate
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+The larger $Staking\:Period$ is, the closer $Effective\:Period$ is to $MaxConsumptionRate$.
+
+A staker achieves the maximum reward for its stake if $Staking\:Period$ = $Minting Period$.
+The reward is:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+Max Reward = \left(MaximumSupply - Supply \right) \times \frac{Stake}{Supply} \times \frac{MaxConsumptionRate}{PercentDenominator}
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+## Delegators Weight Checks
+
+There are bounds set of the maximum amount of delegators' stake that a validator can receive.
+
+The maximum weight $Max\;Weight$ a validator $Validator$ can have is:
+
+<!-- markdownlint-disable MD013 -->
+<!-- vale off -->
+$$
+    Max\;Weight = \min(Validator.Weight \times MaxValidatorWeightFactor, MaxValidatorStake)
+$$
+<!-- vale on -->
+<!-- markdownlint-enable MD013 -->
+
+where $MaxValidatorWeightFactor$ and $MaxValidatorStake$ are the Elastic Subnet
+Parameters described above.
+
+A delegator won't be added to a validator if the combination of their weights
+and all other validator's delegators' weight is larger than $Max\;Weight$. Note
+that this must be true at any point in time.
+
+<!-- vale off -->
+Note that setting $MaxValidatorWeightFactor$ to 1 disables delegation since the $Max\;Weight = Validator.Weight$.
+<!-- vale on -->
+
+
+[^1] Note that `PercentDenominator = 1_000_000` is the denominator used to calculate percentages.
+
+It allows you to specify percentages up to 4 digital positions.
+To denominate your percentage in `PercentDenominator` just multiply it by `10_000`.
+For example:
+
+* `100%` corresponds to `100 * 10_000 = 1_000_000`
+* `1%` corresponds to `1* 10_000 = 10_000`
+* `0.02%` corresponds to `0.002 * 10_000 = 200`
+* `0.0007%` corresponds to `0.0007 * 10_000 = 7`
+
+[^2] Note that $Staking\:Period$ is the staker's entire staking period, not just the
+staker's uptime, that is the aggregated time during which the staker has been
+responsive. The uptime comes into play only to decide whether a staker should be
+rewarded; to calculate the actual reward only the staking period duration is
+taken into account.
```

Comment:
Not sure how to we feel about footnotes. I am not in love with them but seems to me it's easier to read if they are well separated from the points where PercentDenominator and StakingPeriod are referenced

Reply:
Tuned into sections upon Aaron suggestion